e2e-tests/src/checkout.integration.test.js

```diff
@@ -1,81 +1,84 @@
+import { payNotice, acceptCookiePolicy, verifyPaymentAndGetError } from './utils/helpers';
 
-import { payNotice, acceptCookiePolicy, verifyPaymentAndGetError } from "./utils/helpers";
-
-describe("Checkout payment activation tests", () => {
+describe('Checkout payment activation tests', () => {
   /**
    * Test input and configuration
    */
   const CHECKOUT_URL = String(process.env.CHECKOUT_URL);
   const VALID_FISCAL_CODE = String(process.env.VALID_FISCAL_CODE);
-  const INVALID_FISCAL_CODE = String(process.env.INVALID_FISCAL_CODE)
+  const INVALID_FISCAL_CODE = String(process.env.INVALID_FISCAL_CODE);
   const EMAIL = String(process.env.EMAIL);
   const VALID_CARD_DATA = {
     number: String(process.env.CARD_NUMBER),
     expirationDate: String(process.env.CARD_EXPIRATION_DATE),
     ccv: String(process.env.CARD_CCV),
-    holderName: String(process.env.CARD_HOLDER_NAME)
+    holderName: String(process.env.CARD_HOLDER_NAME),
   };
 
-  const VALID_RANGE_END_NOTICE_CODE = Number(String(process.env.VALID_NOTICE_CODE_PREFIX).concat("9999999999999"));
-  const VALID_RANGE_START_NOTICE_CODE = Number(String(process.env.VALID_NOTICE_CODE_PREFIX).concat("0000000000000"));
+  const VALID_RANGE_END_NOTICE_CODE = Number(String(process.env.VALID_NOTICE_CODE_PREFIX).concat('9999999999999'));
+  const VALID_RANGE_START_NOTICE_CODE = Number(String(process.env.VALID_NOTICE_CODE_PREFIX).concat('0000000000000'));
 
   const VALID_NOTICE_CODE = Math.floor(
-    Math.random() * (VALID_RANGE_END_NOTICE_CODE - VALID_RANGE_START_NOTICE_CODE + 1) +
-    VALID_RANGE_START_NOTICE_CODE
+    Math.random() * (VALID_RANGE_END_NOTICE_CODE - VALID_RANGE_START_NOTICE_CODE + 1) + VALID_RANGE_START_NOTICE_CODE,
   ).toString();
-  
-  const PA_IRRAGGIUNGIBILE_NOTICE_CODE = String(process.env.PA_IRRAGGIUNGIBILE_NOTICE_CODE)
+
+  const PA_IRRAGGIUNGIBILE_NOTICE_CODE = String(process.env.PA_IRRAGGIUNGIBILE_NOTICE_CODE);
 
   /**
    * Increase default test timeout (60000ms)
    * to support entire payment flow
-    */
+   */
   jest.setTimeout(60000);
-  jest.retryTimes(3);
+  jest.retryTimes(1);
```

Comment:
Is it right to decrease the number of retries ?👀

Reply:
ops! it's a mistake...